platforms/android/src/jp/wizcorp/phonegap/plugin/WizAssets/WizAssetManager.java

```diff
@@ -96,18 +98,18 @@ public JSONObject getAllAssets() {
         }
         return returnObject;
     }
-    
+
     private void buildDB() {
         // Init DB from bundle assets out to storage
         Log.d(TAG, "Init DB");
-        
+
         // Open the .db file from assets directory
         try {
             InputStream is = that.getAssets().open(DATABASE_INTERNAL_FILE_PATH + DATABASE_NAME);
 
             // Copy the database into the destination
             OutputStream os = new FileOutputStream(DATABASE_EXTERNAL_FILE_PATH + File.separator + DATABASE_NAME);
-        
+
             byte[] buffer = new byte[1024];
```

Comment:
Sounds small yeah, but It does the job... and we use a buffer of the same size for downloading:
https://github.com/Wizcorp/phonegap-plugin-wizAssets/pull/10/files#diff-6d5265fda5622042ce0f322512001a31L243
Doing benchmarking would be great here. I can keep it as a TODO for a later PR.